src/Middleware/Rewrite/src/IISUrlRewriteOptionsExtensions.cs

```diff
@@ -19,7 +19,8 @@ public static class IISUrlRewriteOptionsExtensions
         /// <param name="options">The <see cref="RewriteOptions"/></param>
         /// <param name="fileProvider">The <see cref="IFileProvider"/> </param>
         /// <param name="filePath">The path to the file containing UrlRewrite rules.</param>
-        public static RewriteOptions AddIISUrlRewrite(this RewriteOptions options, IFileProvider fileProvider, string filePath)
+        /// <param name="useNativeIISServerVariables">If the application uses the in-process hosting model, server variables can be sourced natively from IIS. It is disabled by default.</param>
+        public static RewriteOptions AddIISUrlRewrite(this RewriteOptions options, IFileProvider fileProvider, string filePath, bool useNativeIISServerVariables = false)
```

Comment:
I'd invert the bool here and have this on by default. I'm fine with this being a breaking change, however it would be bad if there wasn't a work around.

Reply:
Yeah, that makes sense. Did that in [`9fedfd53`](https://github.com/aspnet/AspNetCore/pull/10022/commits/9fedfd5330a8261cc3e2d970cdf9f32f7c3b11f0).

I was wondering whether to change the default of the parameter or rename it to `disableIISServerVariables`.

I ended up keeping the original name, change the default value and update the associated comment, but let me know if you think renaming the parameter would make more sense to you.